docs/source/markdown/podman-run.1.md

```diff
@@ -826,6 +826,8 @@ Security Options
 
 - **apparmor=unconfined** : Turn off apparmor confinement for the container
 - **apparmor**=_your-profile_ : Set the apparmor confinement profile for the container
+- **mask**=_/path/1:/path/2_: The paths to mask separated by a colon. A masked path
+  cannot be accessed inisde the container.
```

Comment:
```suggestion
  cannot be accessed inside the container.
```

Reply:
and sorry if that was my fat fingers!

Reply:
Fixed, thanks! I should enable spell checker :D